server/TracyWorker.cpp

```diff
@@ -6772,9 +6781,12 @@ void Worker::ProcessContextSwitch( const QueueContextSwitch& ev )
             auto& item = data.back();
             assert( item.Start() <= time );
             assert( item.End() == -1 );
+            //TODO: it may happen that events are being dropped (for example due to breaking in the debugger, or we are simply too slow to handle the events)
+            //      We should handle this properly in some way, but it is unclear how. We can't even really detect it properly other than when cpu doesn't match.
+            assert( item.Cpu() == ev.cpu );
```

Comment:
Yeah, it will happen. This shouldn't be an assert, because some people will run with no asserts and it will be ignored anyways, and other people will complain that the profiler crashes.

Reply:
Should I just remove the assert and ignore the issue or do we actually want to generate a new context switch?
It's not really clear how to fill the data if we want to generate a new cswitch. (what time do we use for end of the previous cswitch, and for start of the new one).

Reply:
It's all guesswork when data is missing. The capture should ideally be able to eventually recover proper state after more data is received.

Reply:
I suppose we could detect dropped events from ETW display it somehow in the UI but it seems out of scope for this PR. Will leave the comment (with a bit more detail if needed) but remove the assert, is that OK with you? (ie, same behavior as before, but documented)

Reply:
It's ok.